artemis-server/src/main/java/org/apache/activemq/artemis/core/protocol/ProtocolHandler.java

```diff
@@ -143,6 +143,8 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception
          }
       }
 
+
+      // the innitial communication happens here...
```

Comment:
hahaha you left this in from the web conf :)

Reply:
ouch.. I will separate this commit and keep it :)
Live coding!!!